hack/generate/csv.sh

```diff
@@ -45,7 +45,7 @@ image "autoscaler-hpa" "${serving}-autoscaler-hpa"
 image "autoscaler-hpa__kube-rbac-proxy"  "$rbac_proxy"
 image "controller"     "${serving}-controller"
 image "controller__kube-rbac-proxy"  "$rbac_proxy"
-image "webhook"        "${serving}-webhook"
+image "webhook__webhook" "${serving}-webhook"
```

Comment:
Had to fully qualify this because otherwise this matches the istio webhook as well and breaks it.

Reply:
good container names, FTW :-)